### PR TITLE
BER-16: Updated the code for UI and functionality

### PR DESCRIPTION
**Pull request by Sonaldev3**
BER-16: Updated the code for UI and functionality
Fixed the linting and other issues

### CODE REVIEW

File: src/app/locate/components/locate-results-list/locate-results-list.component.scss [hunk 1]
```diff
@@ -86,6 +96,8 @@ ul.location-list {
       background: transparent;
       padding: 5px;
       padding-left: 0;
+      text-decoration: none;
+      color: #000;
```
Comment (prashant-pareek) [inline src/app/locate/components/locate-results-list/locate-results-list.component.scss:100]: use scss variable here

File: src/app/locate/locate.module.ts [hunk 2]
```diff
@@ -13,6 +13,12 @@ import { LocateFilterButtonsComponent } from './components/locate-filter-buttons
 import { LocateResultsListComponent } from './components/locate-results-list/locate-results-list.component';
 import { LocationTypePipe } from '../shared/pipes/location-type.pipe';
 import { SafePipe } from '../shared/pipes/safe.pipe';
+import { IconDirectionsComponent } from '../shared/icons/icon-directions/icon-directions.component';
+import { IconEarthComponent } from '../shared/icons/icon-earth/icon-earth.component';
+import { IconEnvelopeComponent } from '../shared/icons/icon-envelope/icon-envelope.component';
+import { IconMapComponent } from '../shared/icons/icon-map/icon-map.component';
+import { IconPhoneComponent } from '../shared/icons/icon-phone/icon-phone.component';
+import { IconUberComponent } from '../shared/icons/icon-uber/icon-uber.component';
```
Comment (prashant-pareek) [inline src/app/locate/locate.module.ts:21]: Icons can be reused so move them to blocks directory

Comment (prashant-pareek) [reply src/app/locate/locate.module.ts:21]: we have existing icons-module

File: src/app/locate/components/locate-results-list/locate-results-list.component.scss (hunk 1 above)
Comment (Sonaldev3) [reply src/app/locate/components/locate-results-list/locate-results-list.component.scss]: Updated

File: src/app/locate/locate.module.ts (hunk 2 above)
Comment (Sonaldev3) [reply src/app/locate/locate.module.ts]: Done.

**Review by prashant-pareek — APPROVED** [review]

**Pull request merged**